### ISSUE TITLE
Add MUSIC_OPEN examples to Preservica Test

### ISSUE THREAD

**Issue by motropuk** [yalelibrary/YUL-DC#1916]
**Story**

Need to add or identify multiple examples in Preservica Test to use for ongoing Preservica development testing. All examples should use the MUSIC_OPEN security tag that both the music admin set login (#1914) and dev team (#1915) will be able to access. 

Examples should represent both data structures available in Preservica V6.  

**Acceptance**
- [x] Add MUSIC_OPEN security tag to Preservica TEST
- [x] Examples with MUSIC_OPEN security tag added/identify in Preservica Test
- [x] Example links documented for use in ongoing development testing work

**Comment by motropuk** [comment]
Structure 1 https://yul-pres-tsdb.library.yale.edu/explorer/explorer.html#browse:SO&bfac21a1-2e80-411a-9504-e784cd16d1e9&null

**Comment by motropuk** [comment]
Asked DPS to add Security Tag to Preservica Test so I can assign this to examples

**Comment by motropuk** [comment]
Waiting on DPS to advise on ingesting an example using structure 2

**Comment by motropuk** [comment]
DPS have hit a snag with this and cant ingest an example using Structure 2 into TEST. So the only Structure 2 example we have is the one in Dev https://preservica-dev-v6.library.yale.edu/explorer/explorer.html#properties:IO&2bb7374a-bbd6-4f8d-a580-5dc2d6eddb02

**Comment by motropuk** [comment]
DPS we add a structure 2 example in time, but it will be a while. So we can either put this on Hold or close this ticket.

**Comment by motropuk** [comment]
Structure 2 example (based on standard output from Preservation digitization service - this time where access and preservation copies exist as a single object which multiple representations) https://yul-pres-tsdb.library.yale.edu/explorer/explorer.html#properties:IO&f6aebc6c-4e05-4d70-b079-97996a6dab4b
